.gitignore

```diff
@@ -5,3 +5,6 @@ Homestead.yaml
 Homestead.json
 Vagrantfile
 .env
+
+/.idea
```

Comment:
`.idea` should probably be in your [global gitignore](https://ryanwinchester.ca/posts/global-gitignore-and-you)